Source/Fuse.Reactive.JavaScript/FuseJS/Observable.js

```diff
@@ -1679,4 +1679,14 @@ Observable._getDataObserver = function(node, key)
 	return res;
 }
 
+Observable.prototype.pull = function(onResult){
+	var self = this;
+	function resolver(obs)
+	{
+		onResult(obs);
```

Comment:
Remove the subscriber first, then call the callback. This avoids an unintentional feedback loop.